### DISCUSSION TITLE
Watchlist feature

### DISCUSSION THREAD

**Discussion by wioo** [seerr-team/seerr#445] [Q&A] [answered]
Is watchlist feature not fully implemented atm? I can star a movie or series but if site is refreshed they aren't starred anymore. Also there isn't any way to see all starred items.

Thanks

**Comment by wioo** [comment, marked as answer]
Let me answer my own question. If one is using latest stable version then there is no link for watchlist page. It is accessible from url bar. Url is http(s)://your_jellyseerr_url/profile/watchlist 

If using develop version, watchlist entries are accessible from profile page.

**Reply by fallenbagel** [reply]
Yup. I did a bunch of watchlist fixes but have not released it yet
